### PR TITLE
 BAEL-5712: Improvement in Listing files under directory

### PR DESCRIPTION
**Pull request by harry9656**

### CODE REVIEW

File: core-java-modules/core-java-io-2/src/main/java/com/baeldung/listfiles/ListFiles.java [hunk 1]
```diff
@@ -27,31 +26,31 @@ public Set<String> listFilesUsingJavaIO(String dir) {
...
         try (Stream<Path> stream = Files.list(Paths.get(dir))) {
             return stream
-                .filter(file -> !Files.isDirectory(file))
-                .map(Path::getFileName)
-                .map(Path::toString)
-                .collect(Collectors.toSet());
+                    .filter(file -> !Files.isDirectory(file))
+                    .map(Path::getFileName)
+                    .map(Path::toString)
+                    .collect(Collectors.toSet());
```
Comment (ashleyfrieze) [inline core-java-modules/core-java-io-2/src/main/java/com/baeldung/listfiles/ListFiles.java:32]: This formatting change looks like it needs reverting

File: core-java-modules/core-java-io-2/src/main/java/com/baeldung/listfiles/ListFiles.java [hunk 2]
```diff
@@ -27,31 +26,31 @@ public Set<String> listFilesUsingJavaIO(String dir) {
...
     public Set<String> listFilesUsingFileWalkAndVisitor(String dir) throws IOException {
         Set<String> fileList = new HashSet<>();
         Files.walkFileTree(Paths.get(dir), new SimpleFileVisitor<Path>() {
             @Override
-            public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IOException {
+            public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) {
                 if (!Files.isDirectory(file)) {
                     fileList.add(file.getFileName()
-                        .toString());
+                            .toString());
```
Comment (ashleyfrieze) [inline core-java-modules/core-java-io-2/src/main/java/com/baeldung/listfiles/ListFiles.java:53]: This formatting change looks like it needs reverting

**Review by ashleyfrieze — CHANGES_REQUESTED** [review]

File: core-java-modules/core-java-io-2/src/main/java/com/baeldung/listfiles/ListFiles.java (hunk 1 above)
Comment (harry9656) [reply core-java-modules/core-java-io-2/src/main/java/com/baeldung/listfiles/ListFiles.java:27]: I double-checked the formatting. The indentation is correct and follows the IntelliJ default formatting for functional code.

File: core-java-modules/core-java-io-2/src/main/java/com/baeldung/listfiles/ListFiles.java (hunk 2 above)
Comment (harry9656) [reply core-java-modules/core-java-io-2/src/main/java/com/baeldung/listfiles/ListFiles.java]: Same as above

**Pull request merged**